docs/PREACT_RECOMMENDATIONS.md

```diff
@@ -1,12 +1,94 @@
 ## Recommendations
 
-Let's look at how to setup recommendations using the Snapfu CLI. See [Getting Started > Setup](https://searchspring.github.io/snap/#/start-setup) for installing Snapfu.
+### Instantiator config
+The instantiators config provides a reference to our template component so that it can be imported at run time.
+
+If you plan to utilize template branching, `instantiators.recommendation.config.branch` must define the current git branch name. Otherwise, a fallback value with the repository default branch name (typically `production`) should be defined.
+
+```typescript
+const config: SnapConfig = {
+    ...
+    instantiators: {
+		recommendation: {
+			components: {
+				DefaultRecommendations: async () => {
+					return (await import('./components/Recommendations/DefaultRecommendations/DefaultRecommendations')).DefaultRecommendations;
+				},
+			},
+
+			config: {
+				branch: BRANCHNAME || 'production',
+			},
+		},
+	},
+}
+```
+
+The `BRANCHNAME` can be defined at runtime via webpack's `DefinePlugin`
+
+```javascript
+const webpack = require('webpack');
+const childProcess = require('child_process');
+const branchName = childProcess.execSync('git rev-parse --abbrev-ref HEAD').toString().trim();
+module.exports = {
+    ...
+    plugins: [
+		new webpack.DefinePlugin({
+			BRANCHNAME: `"${branchName}"`,
+		}),
+	],
+}
+```
+
+## Default Template
+The Searchspring Management Console contains a `Default` template availble for all profiles that does not require the use of the Snapfu CLI to create a custom template. To use the `Default` template, the following instantiator config should be added to your `snap-preact` config.
+
+```js
+instantiators: {
+	recommendation: {
+		components: {
+			Default: async () => {
+				return (await import('./components/Recommendations/Recs')).Recs;
+			},
+		},
+		config: {
+			branch: BRANCHNAME || 'production',
+		},
+	},
+},
+```
+
+Note that the component is not required to be named `Default`, however `instantiators.recommendation.component` must contain the `Default` key as seen in the example above.
+
+Component file: 
+
+```jsx
+import { h } from 'preact';
+import { observer } from 'mobx-react';
+
+import { Recommendation } from '@searchspring/snap-preact-components';
+
+export const Recs = observer((props) => {
+	const controller = props.controller;
+	const store = controller?.store;
+
+	if (!controller.store.loaded && !controller.store.loading) {
+		controller.search();
+	}
+
+	const parameters = store?.profile?.display?.templateParameters;
+
+	return store.results.length > 0 && <Recommendation controller={controller} />;
```

Comment:
^ there is currently a singular templateParameter `title` available for this template that we should be able to plug directly into the `Recommendation` component.

```
<Recommendation controller={controller} title={parameters.title} />
```

We need to ensure this works with the Default template - should be able to test tomorrow.